TensorFlow/LanguageModeling/BERT/run_squad.py

```diff
@@ -940,6 +940,9 @@ def main(_):
   else:
     os.environ["TF_XLA_FLAGS"] = "--tf_xla_enable_lazy_compilation=false"
 
+  # Enable async_io to speed up multi-gpu training with XLA and Horovod.
+  os.environ["TF_XLA_FLAGS"] += "--tf_xla_async_io_level=1"
```

Comment:
no " " separator?
same on line 939

Reply:
You are right. I was misled by line 939. I wonder why it worked?

Reply:
Did some tests. So, the line 939 never worked...

I updated the PR. Thanks @bas-aarts for the catch.

Reply:
looks good